frontend/src/components/TextField/TextField.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { TextField as MuiTextField, BaseTextFieldProps } from '@mui/material';
+
+interface TextFieldProps extends BaseTextFieldProps {
+    children?: React.ReactNode;
+    id: string;
+    label: string;
+    defaultValue?: string;
+    helperText?: string;
+    onChange?: () => void;
```

Comment:
Could you please sort all the props by their requirement, at first required, and only then possible. It will be much more readable.